core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
Some of the tests use _save_exploration instead of update_exploration (editor_test.py, gallery_test.py, exp_services_test.py (of particular import ExplorationSummaryGetTests)), so only updating the contributor ids when update_exploration was called was not sufficient to correctly track contributors (at least for these tests). Does that make sense?